test/e2e/storage/persistent_volumes-gce.go

```diff
@@ -86,13 +86,18 @@ var _ = utils.SIGDescribe("PersistentVolumes GCEPD", func() {
 		ginkgo.By("Initializing Test Spec")
 		diskName, err = e2epv.CreatePDWithRetry()
 		framework.ExpectNoError(err)
+
+		fsType := "ext3"
```

Comment:
if this logic is useful for other tests too, maybe put it in a util function

Reply:
using `GetDefaultFSType` from e2epv